scout/server/blueprints/clinvar/form.py

```diff
@@ -50,7 +50,9 @@ class ClinVarVariantForm(FlaskForm):
         "Inheritance model",
         choices=[("", "-")] + [(item, item) for item in CLINVAR_INHERITANCE_MODELS],
     )
-    clinsig = SelectField("Clinical Significance", choices=[(item, item) for item in CLNSIG_TERMS])
+    clinsig = SelectField(
+        "Clinical Significance", choices=[(item, item) for item in CLNSIG_TERMS[:5]]
```

Comment:
If a user adds several variants in the same submission batch, and sets different submission criteria, how will these be handled by preClinVar? 😊

Reply:
You can't have submissions with more than one criterion, it's one of the changes introduced in the API last month. From the email that they sent to the submitters:


Dear ClinVar API Submitter,

Thank you for using the ClinVar Submission API! We are making changes to the API submission schema to streamline the submission process:

* We will move 'assertionCriteria' to the top level. Note that only one assertion criteria (a citation or a file) may be provided for a submission. This assertion criteria will be applied to all records in the submission.

Reply:
Forgot to add that this is valid also for the submissions via file. PreClinVar handles it fine, in the sense that collects the assertion criteria from the extra args sent by the request, that in Scout are in turn collected from the first variant of the submission [code](https://github.com/Clinical-Genomics/scout/blob/f4b4064591f2888250d0e9aadc42f371017c84c2/scout/server/blueprints/clinvar/controllers.py#L344)

Reply:
For file I'm thinking the user is responsible and can edit their spreadsheet as they wish. 

For api, the first in the sub is quite acceptable. The ideal representation would naturally be to lock the criteria and citation edit plus the select once the first variant is in the sub. That is load the select with data from the current submission if one exists instead of defaulting it to ACMG. But don't worry - I have a feeling for most it will not matter, they will just pick the same. 😸

Reply:
We removed the relative columns from the spreadsheet in #3666, by your suggestion, just saying.. 😆